minutes/2024-05-28.md

```diff
@@ -0,0 +1,139 @@
+# TLA+ Meeting 05/28/2024
+
+MINUTES OF GOVERNING BOARD (FOR PUBLIC RELEASE)
+
+A meeting of the Governing Board of the TLA+ Foundation was held on 28th MAY 2024 at 11:04 am Eastern Time via teleconference.
+
+**Governing Board Members In Attendance**
+
+<table>
+  <tr>
+   <td><strong>TLA+ and Linux Foundation Staff</strong>
+   </td>
+   <td>
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td>Chris Newcombe
+   </td>
+   <td>X
```

Comment:
What do the "X"s indicate? If they mean something, please add it in the header row.

Reply:
Added the Present title. Thanks!